hr_payroll_extended/data/hr.salary.rule.xml

```diff
@@ -0,0 +1,29 @@
+<openerp>
+<data>
+    <record id="hr_payroll.hr_rule_comm" model="hr.salary.rule">
+        <field name="amount_select">code</field>
+        <field name="appears_on_payslip">True</field>
+        <field name="code">COMM</field>
+        <field name="condition_select">python</field>
+        <field name="name">Commissions sur prestations</field>
```

Comment:
This is not a module for the Swiss localization anymore. Please use English for all rules names